### DISCUSSION TITLE
配置好自定义域名后，自定义的域名可以正常访问，但是自定义域名下的关于代理的子页面不挂梯子访问不了

### DISCUSSION THREAD

**Discussion by sz30** [XIU2/CloudflareSpeedTest#308] [讨论]
大神你好，打扰请教一个问题。
配置好自定义域名后，自定义的域名可以正常访问，但是自定义域名下的**关于代理的子页面**不挂梯子访问不了。具体的情况如下：

我在heroku中部署的是一个经过伪装的代理，【xxx.herokuapp.com是伪装界面，xxx.herokuapp.com/ray是代理界面】，

 #189 现在按照教程配置好了CF反代，配置后的自定义的域名是可以访问的（显示的内容和xxx.herokuapp.com一样），即：【 v.with.cf（可以正常访问）关联的是xxx.herokuapp.com（可以正常访问），两者显示的都是伪装页面】

但是真正代理的网址即下属的子页面却无法访问，即：【xxx.herokuapp.com/ray（可以正常访问，显示Bad Request）计划关联的是 v.with.cf/ray（现在不挂梯子访问不了）】，
这种情况可能的原因是什么呢？
[和这个Issue评论中提到的SNI阻断是否是有关系呢？](https://github.com/XIU2/CloudflareSpeedTest/issues/217#issuecomment-1135744087)
提出Issue过了俩小时，刚又可以了，跟这个Issue里面的描述特别像。。 #217

**Comment by XIU2** [comment]
因为我不玩这类代理玩法，所以我无法给予你帮助。。。

**Comment by pchaos** [comment]
感觉已经废了。我换hysteria了

**Comment by fredericky123** [comment]
> 感觉已经废了。我换hysteria了

好用吗 hysteria有什么好的客户端

**Comment by pchaos** [comment]
我是手工配置的。我感觉配置比小火箭简单

fredericky123 ***@***.***> 于2022年6月12日周日 13:17写道：

**Comment by fredericky123** [comment]
> 我是手工配置的。我感觉配置比小火箭简单 fredericky123 ***@***.***> 于2022年6月12日周日 13:17写道：
> […](#)
> 感觉已经废了。我换hysteria了 好用吗 hysteria有什么好的客户端 — Reply to this email directly, view it on GitHub <[#220 (comment)](https://github.com/XIU2/CloudflareSpeedTest/issues/220#issuecomment-1153074801)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADR7ND6VOC3IVICUKCJRLMTVOVXFRANCNFSM5YBWIFQQ> . You are receiving this because you commented.Message ID: ***@***.***>

会不会暴露服务器ip了 怎么能避免

**Comment by pchaos** [comment]
暴露好久了，好像也没事。可能还没大规模人员使用。

fredericky123 ***@***.***> 于2022年6月13日周一 17:41写道：

**Comment by pchaos** [comment]
我是多开几个端口，轮换着端口用

pchaos ***@***.***> 于2022年6月13日周一 17:42写道：

**Comment by crazypeace** [comment]
你添加路由的时候，有没有写  `v.with.cf/*` ? 还是说，写的 `v.with.cf` ?
